test/Command/CommandMetadataEnricherTest.php

```diff
@@ -14,26 +14,52 @@
 namespace Broadway\Bundle\BroadwayBundle\Command;
 
 use Broadway\Domain\Metadata;
+use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Event\ConsoleCommandEvent;
+use Symfony\Component\Console\Input\ArgvInput;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class CommandMetadataEnricherTest extends TestCase
 {
+    /**
+     * @var MyCommand
+     */
     private $command;
```

Comment:
we might as well use typed properties here now that we are on PHP 7.4

Reply:
I was temped to do so, but I don't want to change things that are not needed as the PR is already huge.

Reply:
Good call!